src/jit/compiler.hpp

```diff
@@ -105,7 +105,14 @@ template<typename T>
 inline
 T    genFindLowestBit(T value)
 {
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable:4146) // unary minus operator applied to unsigned type, result still unsigned
+#endif
     return (value & -value);
```

Comment:
It seems that instead of the warnings disabling, it would be cleaner to use (value & (~value + 1)), where both operands are unsigned and no warning is issued.